cnf-certification-test/networking/netcommons/netcommons.go

```diff
@@ -144,3 +146,38 @@ func FilterIPListByIPVersion(ipList []string, aIPVersion IPVersion) []string {
 	}
 	return filteredIPList
 }
+
+func FindRogueContainersListeningToPorts(containers []*provider.Container, portsToTest map[int32]bool) []string {
```

Comment:
this function doesn't test the port that are really listening right? 
FindRogueContainersListeningToPorts -> FindRogueContainersDeclaredListeningToPorts

Reply:
Fixed.

Reply:
Correct, declared ports.